spec/unit/actions/synchronize_user_filters_spec.rb

```diff
@@ -143,4 +143,36 @@
       end
     end
   end
+
+  context 'when using sync_multiple_projects_based_on_custom_id mode' do
+    let(:params) do
+      params = {
+        GDC_GD_CLIENT: client,
+        input_source: 'foo',
+        domain: 'bar',
+        filters_config: { labels: [] },
+        multiple_projects_column: 'id_column',
+        sync_mode: 'sync_multiple_projects_based_on_custom_id',
+        gdc_logger: logger
+      }
+      GoodData::LCM2.convert_to_smart_hash(params)
+    end
+    let(:CSV) { double('CSV') }
+
+    before do
+      allow(project).to receive(:metadata).and_return(
+        'GOODOT_CUSTOM_PROJECT_ID' => 'project-123'
+      )
+      allow(project).to receive(:uri).and_return('project-uri')
+      allow(project).to receive(:add_data_permissions)
+      allow(domain).to receive(:clients).and_return([])
+      allow(data_source).to receive(:realize).and_return('filepath')
+    end
+
+    it 'fails if the MUF set is empty' do
+      expect(File).to receive(:open)
+      expect(CSV).to receive(:foreach)
+      expect { subject.class.call(params) }.to raise_error /The filter set can not be empty/
```

Comment:
Lint/AmbiguousRegexpLiteral: Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.